### DISCUSSION TITLE
[Question] Is it possible to specify a BaseModel class as a Redoc Model?

### DISCUSSION THREAD

**Discussion by jsmoliak** [0b01001001/spectree#386] [Q&A]
In the [Redoc Interactive Demo](http://redocly.github.io/redoc/), specific data models can be categorized under the "MODELS" tag. Does spectree provide a method to pull in / mark a BaseModel class as a model for redoc to illustrate?

See [The Pet Model](http://redocly.github.io/redoc/#tag/pet_model) as an example.

![image](https://user-images.githubusercontent.com/15694939/200969176-cfb046a8-8843-4755-840a-c030b5fd5e9c.png)

**Comment by kemingy** [comment]
I guess it's https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/.

But I didn't find this in the OpenAPI doc. (correct me if I'm wrong)

For now, you can find the information if you're using swagger.
